print/resources/inkstitch.js

```diff
@@ -397,6 +414,118 @@ $(function() {
     $(this).addClass("active");
   });
 
+  // Footer
+  function getEditMode(){
+    return $('#switch-mode').prop('checked');
+  }
+
+  $('#switch-mode').change(function() {
+    var editMode = getEditMode();
+    if (editMode) {
+      $('#footer-info-text').text( $('#footer-info-text' ).html());
+      $('#tool-bar .edit-only').prop("disabled", true);
+    } else {
+      $('#footer-info-text').css('display', 'block');
+      var sourceText = $('#footer-info-text').text();
+      $('#footer-info-text').html( sourceText );
+      $('#tool-bar .tb-button.edit-only').prop('disabled', false);
+    }
+  });
+
+  function updateFooter() {
+    var editMode = getEditMode();
+    var footerText = '';
+    if (editMode) {
+      footerText = $('#footer-info-text' ).text();
+    } else {
+      footerText = $('#footer-info-text').html();
+    }
+    $('.footer-info').html(footerText);
+    var content = $('.footer-info').html();
+    $.postJSON('/settings/footer-info', {value: content});
+  }
+
+  function formatText(selection, value) {
+    var htmlMode = getEditMode();
+    if(!htmlMode) {
+      if(window.getSelection().toString()){
+        document.execCommand(selection, false, value);
+        updateFooter();
+      }
+    }
+  }
+
+  $('#tb-bold').click(function(selection) {
+    formatText('bold');
+  });
+
+  $('#tb-italic').click(function() {
+    formatText('italic');
+  });
+
+  $('#tb-underline').click(function() {
+    formatText('underline');
+  });
+
+  $('#tb-remove').click(function() {
+    formatText('removeFormat');
+  });
+
+  $('#tb-reset').click(function() {
+    var htmlMode = getEditMode();
+    if(htmlMode) {
+      $('#footer-info-text').text('<div>Proudly generated with <a href="http://inkstitch.org/" target="_blank">Ink/Stitch</a></div>');
```

Comment:
Unfortunately, this won't work for non-English speakers. :(

When I had other similar situations, I put the text into a hidden div (dispay: none) in the jinja2 template, which allowed me to translate the text.  Then in the javascript, I fetched it out of the div and inserted it where I actually wanted it.

Can we do that here?